example/android/build.gradle

```diff
@@ -32,7 +32,6 @@ allprojects {
         }
 
         google()
-        jcenter()
```

Comment:
hey, thanks for submitting this PR!
From my testing this change breaks the example app build. Let me know if its different for you.

Since this file is actually something generated by expo, I think we can skip patching it ourselves and revert changes to it in this PR. Eventually in different PR we will upgrade the expo sdk for the example app where I expect this to be patched by the expo team 👍

Reply:
Ok, reverted. I'll try to update the example in another PR.

Reply:
I've tackled the upgrade in #52 
Thanks for your help